### PR TITLE
Chip component in Sparkle

### PR DESCRIPTION
**Pull request by philipperolet**
As presented in the assistant responses in figma (cf screenshot)
image

Related discussion [slack thread](https://dust4ai.slack.com/archives/C050SM8NSPK/p1694595377321409)

### CODE REVIEW

**Comment by philipperolet** [comment]
Published sparkle for testing on  ``0.1.60-pr-chip-alpha``

File: sparkle/src/components/Chip.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React, { MouseEvent, ReactNode } from "react";
+
+import { SparkleContext } from "@sparkle/context";
+import { classNames } from "@sparkle/lib/utils";
+
+type ChipProps = {
+  onClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+  size?: "sm" | "md";
```
Comment (Duncid) [inline sparkle/src/components/Chip.tsx:8]: Let's not add props that we don't use yet!

File: sparkle/src/components/Chip.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  color?: "emerald" | "amber" | "slate" | "violet";
+  label?: string;
+  children?: ReactNode;
+  className?: string;
+  href?: string;
```
Comment (Duncid) [inline sparkle/src/components/Chip.tsx:13]: Why href?

**Review by spolu — COMMENTED** [review]
unused import :)

**Comment by philipperolet** [comment]
Visual on storybook
image

**Review by Duncid — DISMISSED** [review]

File: sparkle/src/components/Chip.tsx (hunk 2 above)
Comment (philipperolet) [reply sparkle/src/components/Chip.tsx:13]: ok I removed onclick & href

**Pull request merged**